### PR TITLE
Document data-standard tests

### PR DESCRIPTION
**Pull request by rhiaro**
Documentation for what the tests in the data standard repo do, how and why. #8 can be closed when this is merged.

### CODE REVIEW

File: testing.md [hunk 1]
```diff
@@ -46,10 +57,65 @@ To run all the tests:
...
+
+```
+[
+    {
+        "statementId": "2f7bf9370f1254068e5e946df067d07d",
+        "declarationSubject": "xyz",
+        "recordId": "123",
+        "recordType": "entity",
+        "recordDetails": {
+            "entityType": "unknownEntity",
```
Comment (BibianaC) [inline testing.md:108]: Wouldn't this only be the case with "entity" statements?

File: testing.md [hunk 2]
```diff
@@ -46,10 +57,65 @@ To run all the tests:
...
+            "isComponent": False,
+        },
+    }
+]
+```
+
+Start from this, and add only the field you are testing. If you are testing a field in a nested object (eg. `publicationDetails/publisher/name`) you may need to add more data to cover additional required fields (eg. `publicationDetails/publicationDate`). Check the schema itself to find out which fields are required for the various components.
```
Comment (kd-ods) [inline testing.md:115]:
```suggestion
Start from a minimal statement like this, and add only the field you are testing. If you are testing a field in a nested object (eg. `publicationDetails/publisher/name`) you may need to add more data to cover additional required fields (eg. `publicationDetails/publicationDate`). Check the schema itself to find out which fields are required for the various objects.
```

File: testing.md [hunk 3]
```diff
@@ -46,10 +57,65 @@ To run all the tests:
...
+
+After adding new files make sure **run the tests** (`pytest tests/test_data.py`) to check they pass.
```
Comment (kd-ods) [inline testing.md:117]:
```suggestion
After adding new files make sure to **run the tests** (`pytest tests/test_data.py`) to check they pass.
```

**Review by kd-ods — CHANGES_REQUESTED** [review]
A few suggestions.

**Review by kd-ods — APPROVED** [review]
Great.

**Pull request merged**